docs/plugins/export.rst

```diff
@@ -36,11 +39,18 @@ The ``export`` command has these command-line options:
 
 * ``--append``: Appends the data to the file instead of writing.
 
+* ``--format`` or ``-f``: Specifies the format the data will be exported as. If not informed, JSON will be used by default.
+  For example::
+
+      $ beet export -f csv beatles
+      $ beet export -f json beatles
+      $ beet export -f xml beatles
```

Comment:
I think we can do without the full command-line examples here. Can we just list the options?

Reply:
Sounds good